modules/simplesamlphp.nix

```
@@ -42,6 +42,9 @@ let
   configFile = pkgs.writeText "config.php" ''
     <?php
 
+    $adminpassword = file('${cfg.adminPasswordFile}', FILE_IGNORE_NEW_LINES)[0];
+    $secretsalt = file('${cfg.secretSaltFile}', FILE_IGNORE_NEW_LINES)[0];
+
```

will not return `null` if file is empty. also probably more idiomatic:
```suggestion
    $adminpassword = trim(file_get_contents('${cfg.adminPasswordFile}'));
    $secretsalt = trim(file_get_contents('${cfg.secretSaltFile}'));

```